cmd2/py.typed

```diff
@@ -0,0 +1 @@
+# PEP 561
```

Comment:
So what does adding this file with this comment do?

Reply:
The comment doesn't matter. But the file is what's defined in PEP-561 to signal that we support type annotations. Without this, any project that depends on us will get a bunch of warnings and errors using mypy because it'll assume everything from our package is coming in as `Any`.